browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -442,6 +450,8 @@ void RewardsDOMHandler::OnGetContentSiteList(std::unique_ptr<brave_rewards::Cont
       publishers->Append(std::move(publisher));
     }
 
+    // numExcludedSites should update with the new content list
+    GetNumExcludedSites();
```

Comment:
Why do we need to get new number list when we are fetching regular list?

Shouldn't we just get it when you do exclude/include action?